.prow.yaml

```diff
@@ -0,0 +1,45 @@
+presubmits:
+  - name: kustomize-build
+    decorate: true
+    max_concurrency: 1
+    run_if_changed: ".*yaml"
+    skip_report: false
+    context: aicoe-ci/prow/kustomize-build
+    spec:
+      containers:
+        - image: quay.io/operate-first/opf-toolbox:v0.3.1
+          command:
+            - "test-kustomize-build"
+          resources:
+            requests:
+              memory: "256Mi"
+  - name: conftest
+    decorate: true
+    max_concurrency: 1
+    run_if_changed: ".*yaml"
+    skip_report: false
+    context: aicoe-ci/prow/conftest
+    spec:
+      containers:
+        - image: quay.io/operate-first/opf-toolbox:v0.3.1
+          command:
+            - "test-conftest"
```

Comment:
is this to run from the image? if so, isn't the file kept at `/usr/local/bin/` ?